2021-04-20 13:48:19,712 INFO    MainThread:2974 [wandb_setup.py:_flush():69] setting env: {}
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210420_134819-1k3c9atf/logs/debug.log
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210420_134819-1k3c9atf/logs/debug-internal.log
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_init.py:init():369] calling init triggers
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-20 13:48:19,713 INFO    MainThread:2974 [wandb_init.py:init():418] starting backend
2021-04-20 13:48:19,713 INFO    MainThread:2974 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-20 13:48:19,724 INFO    MainThread:2974 [backend.py:ensure_launched():123] starting backend process...
2021-04-20 13:48:19,728 INFO    MainThread:2974 [backend.py:ensure_launched():128] started backend process with pid: 2992
2021-04-20 13:48:19,730 INFO    MainThread:2974 [wandb_init.py:init():423] backend started and connected
2021-04-20 13:48:19,731 INFO    MainThread:2974 [wandb_init.py:init():465] updated telemetry
2021-04-20 13:48:19,732 INFO    MainThread:2974 [wandb_init.py:init():484] communicating current version
2021-04-20 13:48:20,507 INFO    MainThread:2974 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-20 13:48:20,508 INFO    MainThread:2974 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-20 13:48:20,719 INFO    MainThread:2974 [wandb_init.py:init():522] starting run threads in backend
2021-04-20 13:48:20,743 INFO    MainThread:2974 [wandb_run.py:_console_start():1524] atexit reg
2021-04-20 13:48:20,744 INFO    MainThread:2974 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-20 13:48:20,745 INFO    MainThread:2974 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-20 13:48:20,748 INFO    MainThread:2974 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-20 13:48:20,748 INFO    MainThread:2974 [wandb_init.py:init():546] run started, returning control to user process
2021-04-20 13:48:23,608 INFO    MainThread:2974 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-20 13:48:23,610 INFO    MainThread:2974 [wandb_run.py:_restore():1466] restore
2021-04-20 13:48:25,827 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 545
  total_bytes: 545
}

2021-04-20 13:48:26,735 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 545
  total_bytes: 3376
}

2021-04-20 13:48:27,108 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,210 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,313 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,415 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,518 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,620 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:27,722 INFO    MainThread:2974 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3376
  total_bytes: 3376
}

2021-04-20 13:48:28,803 INFO    MainThread:2974 [wandb_run.py:_show_files():1838] logging synced files
